Document metric name restrictions (#1205)
grafana/k6-docs: src/data/markdown/translated-guides/en/02 Using k6/02 Metrics.md

In that output, all the metrics that start with `http`, `iteration`, and `vu` are _built-in_ metrics, which get written to stdout at the end of a test.
For details of all metrics, refer to the [Metrics reference](/using-k6/metrics/reference/).

### Metric name restrictions

<Blockquote mod="warning" title="">

> In the future metrics names will be further limited to comply with Open telemetry and Prometheus limitations.

Reviewer: ```suggestion
In the future, metrics names will be further limited to comply with OpenTelemetry and Prometheus limitations.
```

> That will mean that metric names can be 1 to 63 symbols of ASCII letters, numbers and `_`. And must not start with a number.

Reviewer: ```suggestion
That means metric names can be 1 to 63 symbols of ASCII letters, numbers, or  `_`. Metric names must not start with a number.
```

</Blockquote>

> At the moment a metric name can be 1 to 128 symbols of:

Reviewer: ```suggestion
At the moment, a metric name can be 1 to 128 symbols of:
```
